app/src/main/java/com/onelist/dto/Item.kt

```diff
@@ -1,8 +1,8 @@
 package com.onelist.dto
 
-data class Item(var itemID: Int, var name: String, var categoryID: Int, var quantity : Int) {
+data class Item(var itemId: Int, var name: String, var categoryId: Int, var quantity : Int) {
```

Comment:
It is better to follow variable naming conventions stated by kotlin which i changed theirs to match

Reply:
See notes above.

Enhance this by describing the attributes, above and beyond what can be given by context.